frontend/src/data/Game.ts

```diff
@@ -0,0 +1,13 @@
+export class Game {
```

Comment:
In den meisten Fällen, insbesondere, wenn du keine Logik in deinen Objekten hast, würde ich dir eher dazu raten auf `type` statt `class` zurückzugreifen.